libgit.el

```diff
@@ -32,6 +32,11 @@
 (unless module-file-suffix
   (error "Module support not detected, libgit can't work"))
 
+(defcustom libgit-cmake-generator nil
+  "CMake generator to use."
+  :type 'string
```

Comment:
It can, and by default is, `nil`. Also what would be a good non-nil value? Is there more than one non-nil value that even makes sense. Either way, a good non-nil value should be provided as a choice, that the user can select without having to learn more about Cmake first.